portal/server/api.js

```diff
@@ -185,7 +185,7 @@ const updateDealName = async (id, newName, token) => {
  * @param {Object} statusUpdate - An object containing the `_id` property representing
  * the deal ID and the `status` property representing the new status of the deal.
  * @param {string} token - A token used for authorization in the API call.
- * @returns {Object|boolean} - An object containing the `status` code and the `data` from the API response, or `false` if the `_id` is not valid.
+ * @returns {Promise<Object|boolean>} - An object containing the `status` code and the `data` from the API response, or `false` if the `_id` is not valid.
```

Comment:
Nit but I think formatting rules suggest that this should be `Promise<object | boolean>`